lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -693,7 +693,7 @@ sub upgrade {
     for my $check (_applicable_upgrade_tests($dbinfo)) {
         my $sth = $request->{dbh}->prepare($check->test_query);
         $sth->execute()
-            or die "Failed to execute pre-migration check " . $check->name;
+            or die "Failed to execute pre-migration check " . $check->name . ", " . $sth->errstr;
```

Comment:
This explains more in detail what is happening. I think it is justified for developers and migrators.